mw-webapp/src/component/footer/Footer.module.scss

```diff
@@ -1,36 +1,44 @@
 @import "src/styles/variables";
 
-$userNameFontSize: 16px;
+$logoHeight: 55px;
+$privacyPaddingTop: 10px;
+$privacyPaddingBottom: 12px;
+$iconSize: 32px;
+$privacyLineHeight: 140%;
 
 .footer {
   background-color: var(--quaternaryBgColor);
+  margin-inline: $paddingMedium;
```

Comment:
1. Why do we need margin left and right for footer? If we need then lets use common margin instead of margin-inline.
2.Margin is not padding, so var is wrong

Reply:
Using this margin I solve one problem: when the screen's width is reduced (less than 1560px), the logo and social media block with icons are glued to the sides of the screen. I added them space to "breathe".
<img src="https://github.com/tritonJS826/masters-way/assets/8752900/f7a59463-ddf2-4709-911c-f1915e4768a8" width="500" />

Actually, I found the bug here, I will try to use here another approach.

Reply:
Не смогла повторить , вроде ничего не прижимается на демо. Давай избегать любых марджинов, если нужно, то лучше использовать паддинги
![Screenshot 2024-06-24 22:22:39](https://github.com/tritonJS826/masters-way/assets/57002521/a6a0777c-ad56-41d5-94ae-2e12c3c02198)

Reply:
Прижимается при моей реализации)
В демо отсуп есть (не прижимается), но линейка не выровнена с логотипом и "Политикой конфиденциальности", как по дизайну

Reply:
I fixed the bug, 'inline' and 'block' are gone